packages/commonwealth/server/routes/createChain.ts

```diff
@@ -52,9 +55,11 @@ export const Errors = {
   InvalidGithub: 'Github must begin with https://github.com/',
```

Comment:
Going forward, can we handle this for them? (ie if they give us a @handle, we prefix it with the URI and try that.)